app/views/requests/show.html.erb

```diff
@@ -65,7 +65,11 @@
               <tr>
                 <th>Item</th>
                 <th>Quantity</th>
-                <th>Fulfillment Location Inventory</th>
+                <% default_storage_location = @request.organization.default_storage_location ||
+                                              @request.partner.default_storage_location_id %>
```

Comment:
I found one more thing on my last go-through -- and this is *extremely* nitpicky.
I think it would be better if we switched the two clauses in this or?

Why?  You are only using the existence of a default storage location,  but, If someone later decides to use the default storage location (say we decide to change the title) - it should match what we are using for getting the on_hand_for_location.    In the probably rare case where the organization and partner both have default locations,  we use the partner location.

Reply:
Sure. I fixed it here [9e77b48](https://github.com/rubyforgood/human-essentials/pull/4466/commits/9e77b48ac7cc370c3f89ed71f4a85a4f6ce0ab81)